conda-store-server/conda_store_server/_internal/schema.py

```diff
@@ -154,10 +154,9 @@ class BuildArtifactType(str, enum.Enum):
     LOGS = "LOGS"
     YAML = "YAML"
     CONDA_PACK = "CONDA_PACK"
-    DOCKER_BLOB = "DOCKER_BLOB"
     DOCKER_MANIFEST = "DOCKER_MANIFEST"
-    CONTAINER_REGISTRY = "CONTAINER_REGISTRY"
     CONSTRUCTOR_INSTALLER = "CONSTRUCTOR_INSTALLER"
+    _ = "CONTAINER_REGISTRY"
```

Comment:
Users might have `CONTAINER_REGISTRY` as part of their settings (persisted to the database) for `build_artifacts_kept_on_deletion`. These are only updated by admins in the admin ui. They are not updated when a new conda_store config is pushed to the server (see [ensure_settings](https://github.com/conda-incubator/conda-store/blob/main/conda-store-server/conda_store_server/app.py#L507) function). 

If the user has a build artifact type that is not in this list, then the server will error. So, we'll leave this as a valid type.

Reply:
Good idea.